no/Mortgage/v0/schema/schema.yaml

```diff
@@ -0,0 +1,784 @@
+              min: 0
+              description: The estimated value of the property, in NOK
+            propertyAddress:
+              title: Property address
+              description: An address for a real-estate property
+              type: object
+              additionalProperties: false
+              required:
+                - streetAddress
+                - city
+                - postalCode
+              properties:
+                streetAddress:
+                  type: string
+                  minLength: 1
+                city:
+                  type: string
+                  minLength: 1
+                postalCode:
+                  type: string
+                  description: A valid postal code, with exactly four digits
+                  pattern: "\\d{4}"
+            propertyType:
+              type: string
+              title: The property type
+              enum:
+                - HOUSE
+                - TERRACED_HOUSE
+                - VACATION_HOME
+                - APARTMENT_SELF_OWNED
+                - APARTMENT_COOPERATIVE
+                - OTHER
+              "meta:enum":
+                HOUSE: An ordinary house with a plot (no "hus")
+                TERRACED_HOUSE: A house that is built adjacent to other houses (no "rekkehus")
+                VACATION_HOME: |
+                  A house built for recreational living that may or may not
+                  be fit as a permanent resident (no "fritidsbolig")
+                APARTMENT_COOPERATIVE: |
+                  An apartment that belongs to condominium compound (no "andel i borettslag")
+                APARTMENT_SELF_OWNED: An apartment that is owned directly by the applicant (no "leilighet")
+                OTHER: |
+                  Some other form of real-estate property that does not belong
+                  to any of the above categories
+            squareMeters:
+              type: integer
+              min: 0
+              description: The number of square meters of the property (excluding any plot)
+            existingMortgage:
+              type: integer
+              min: 0
+              description: How mortgaged the property is, in NOK
+            monthlyCost:
+              type: integer
+              min: 0
+              description: |
+                The sum of all montlhy costs associated with the property, such as
+                upkeep or a monthly fee to a condominium compound if the property
+                is an apartment
+            condominiumCompoundDebt:
+              type: integer
+              min: 0
+              title: Debt to the condominium compound
+              description: |
+                The debt to the condominium compound that belongs to the property,
+                if the property belongs to a condominium compound.
+                This value should not be given if the property
+                is of another property type than APARTMENT_COOPERATIVE.
+            defaulted:
```

Comment:
Would it be appropriate to add a description to this? I don't personally recognise the term 'defaulted' and maybe others will need some future clarifications as well.

Reply:
Hm, yes you right, this should probably be explained a bit more.

Reply:
I have added a description now. I hope it works?

Reply:
@6185541513 are you satisfied? :)